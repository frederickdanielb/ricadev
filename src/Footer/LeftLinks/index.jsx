import LeftLinks from "./LeftLinks";

export default LeftLinks