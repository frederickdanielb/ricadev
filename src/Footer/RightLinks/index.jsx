import RightLinks from "./RightLinks";

export default RightLinks