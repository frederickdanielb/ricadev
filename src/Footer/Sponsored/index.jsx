import Sponsored from "./Sponsored";

export default Sponsored