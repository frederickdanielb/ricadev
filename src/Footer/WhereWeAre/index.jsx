import WhereWeAre from "./WhereWeAre";

export default WhereWeAre