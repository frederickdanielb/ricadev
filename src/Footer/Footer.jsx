import ContactUs from "./ContactUs";
import LeftLinks from "./LeftLinks";
import RightLinks from "./RightLinks";
import Sponsored from "./Sponsored";
import SubFooter from "./SubFooter";
import Topics from "./Topics";
import WhereWeAre from "./WhereWeAre";

const Footer = ({ config }) => {
	return (
		<footer>
			<div className="footer section-b-space section-t-space">
				<div className="container">
					<div className="row order-row">
						<ContactUs logoUrl={config.page.logo_url} contact={config.contact} />
						<LeftLinks links={config.page.footer.left_links} />
						<WhereWeAre />
						<RightLinks links={config.page.footer.right_links} />
						<Topics topics={config.page.footer.topics} />
						<Sponsored logos={config.page.footer.partners_logos}/>
					</div>
				</div>
			</div>
			<SubFooter />
		</footer>
	);
};
export default Footer;
