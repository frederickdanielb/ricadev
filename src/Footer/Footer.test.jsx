import {
	render,
	fireEvent,
	waitFor,
	screen,
	act,
} from "@testing-library/react";
import { MockConfig } from "../../config/_mocks_/config";
import Footer from "./Footer";

describe("loader component", () => {
	let container = render(<Footer config={MockConfig}/>);
	it("should render correctly", async () => {
		expect(container.asFragment()).toMatchSnapshot();
	});
});
