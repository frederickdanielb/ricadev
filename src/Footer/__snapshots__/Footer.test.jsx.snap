// Vitest Snapshot v1

exports[`loader component > should render correctly 1`] = `
<DocumentFragment>
  <footer>
    <div
      class="footer section-b-space section-t-space"
    >
      <div
        class="container"
      >
        <div
          class="row order-row"
        >
          <div
            class="col-xl-2 col-md-6 order-cls"
          >
            <div
              class="footer-title mobile-title"
            >
              <h5>
                contact us
              </h5>
            </div>
            <div
              class="footer-content"
            >
              <div
                class="contact-detail"
              >
                <div
                  class="footer-logo"
                >
                  <img
                    alt=""
                    class="img-fluid lazyload"
                    src="https://www.panamericanaturismo.cl/_ns_img/logo/logoPanamericana200x70.png"
                  />
                </div>
                <ul
                  class="contact-list"
                >
                  <li>
                    <i
                      class="fas fa-map-marker-alt"
                    />
                     Av. Los Leones 1071
                  </li>
                  <li>
                    <i
                      class="fas fa-phone-alt"
                    />
                     +562 2 476 5800
                  </li>
                  <li>
                    <i
                      class="fas fa-envelope"
                    />
                     test@test.cl
                  </li>
                </ul>
              </div>
            </div>
          </div>
          <div
            class="col-xl-2 col-md-3"
          >
            <div
              class="footer-space"
            >
              <div
                class="footer-title"
              >
                <h5>
                  Destinos
                </h5>
              </div>
              <div
                class="footer-content"
              >
                <div
                  class="footer-links"
                >
                  <ul>
                    <li>
                      <a
                        href="https://www.panamericanaturismo.cl/Destinos/?/Zona/3/Caribe"
                      >
                        Caribe
                      </a>
                    </li>
                    <li>
                      <a
                        href="https://www.panamericanaturismo.cl/Destinos/?/Zona/9/Centro-America"
                      >
                        Centro América
                      </a>
                    </li>
                    <li>
                      <a
                        href="https://www.panamericanaturismo.cl/Destinos/?/Zona/2/Sudamerica"
                      >
                        Sudamérica
                      </a>
                    </li>
                    <li>
                      <a
                        href="https://www.panamericanaturismo.cl/Destinos/?/Zona/10/Chile"
                      >
                        Chile
                      </a>
                    </li>
                    <li>
                      <a
                        href="https://www.panamericanaturismo.cl/Destinos/?/Zona/13/Europa"
                      >
                        Europa
                      </a>
                    </li>
                    <li>
                      <a
                        href="https://www.panamericanaturismo.cl/Destinos/?/Zona/19/Destinos-Lejanos"
                      >
                        Destinos Lejanos
                      </a>
                    </li>
                    <li>
                      <a
                        href="https://www.panamericanaturismo.cl/Destinos/?/Zona/1/USA"
                      >
                        Norteamérica
                      </a>
                    </li>
                  </ul>
                </div>
              </div>
            </div>
          </div>
          <div
            class="col-xl-3 col-md-6"
          >
            <div
              class="footer-title"
            >
              <h5>
                Dónde estámos
              </h5>
            </div>
            <div
              class="footer-content"
            >
              <div
                class="footer-map"
              >
                <iframe
                  allowfullscreen=""
                  src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3329.768936005934!2d-70.60544108480117!3d-33.42926798078003!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x9662cf72251d4829%3A0xe2cfa9b2f844f83c!2sAv.%20Los%20Leones%201071%2C%20Providencia%2C%20Regi%C3%B3n%20Metropolitana!5e0!3m2!1ses-419!2scl!4v1647817812809!5m2!1ses-419!2scl"
                  title="google-map"
                />
              </div>
            </div>
          </div>
          <div
            class="col-xl-2 col-md-3 order-cls"
          >
            <div
              class="footer-space"
            >
              <div
                class="footer-title"
              >
                <h5>
                  Sobre nosotros
                </h5>
              </div>
              <div
                class="footer-content"
              >
                <div
                  class="footer-links"
                >
                  <ul>
                    <li>
                      <a
                        href="https://www.panamericanaturismo.cl/Cotiza-Tu-Viaje"
                      >
                        Cotiza tu viaje
                      </a>
                    </li>
                    <li>
                      <a
                        href="https://www.panamericanaturismo.cl/Agente-Especial"
                      >
                        Agente especial
                      </a>
                    </li>
                    <li>
                      <a
                        href="https://www.panamericanaturismo.cl/Politicas-y-Condiciones/Politicas-de-Privacidad/"
                      >
                        Políticas de Privacidad
                      </a>
                    </li>
                    <li>
                      <a
                        href=""
                      >
                        Condiciones y Términos de Uso
                      </a>
                    </li>
                    <li>
                      <a
                        href="https://www.panamericanaturismo.cl/_pdf/20170731_Politica_explotacion_sexual_en_turismo.pdf"
                      >
                        Política sobre la explotación sexual
                      </a>
                    </li>
                  </ul>
                </div>
              </div>
            </div>
          </div>
          <div
            class="col-xl-3 col-md-6"
          >
            <div
              class="footer-title"
            >
              <h5>
                Sponsored
              </h5>
            </div>
            <div
              class="footer-content"
            >
              <img
                alt=""
                class="img-fluid lazyload"
                src="https://www.panamericanaturismo.cl/_ns_img/footer/logo-web-pay-plus.png"
                width="200"
              />
              <img
                alt=""
                class="img-fluid lazyload"
                src="https://www.panamericanaturismo.cl/_ns_img/footer/logo-iata-achet.png"
                width="200"
              />
              <img
                alt=""
                class="img-fluid lazyload"
                src="https://www.tsyacom.cl/images/logo.png"
                width="200"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="sub-footer"
    >
      <div
        class="container"
      >
        <div
          class="row "
        >
          <div
            class="col-xl-6 col-md-6 col-sm-12"
          >
            <div
              class="footer-social"
            >
              <ul>
                <li>
                  <a
                    href="/"
                  >
                    <i
                      class="fab fa-facebook-f"
                    />
                  </a>
                </li>
                <li>
                  <a
                    href="/"
                  >
                    <i
                      class="fab fa-instagram"
                    />
                  </a>
                </li>
                <li>
                  <a
                    href="/"
                  >
                    <i
                      class="fab fa-twitter"
                    />
                  </a>
                </li>
                <li>
                  <a
                    href="/"
                  >
                    <i
                      class="fab fa-google"
                    />
                  </a>
                </li>
              </ul>
            </div>
          </div>
          <div
            class="col-xl-6 col-md-6 col-sm-12"
          >
            <div
              class="copy-right"
            >
              <p>
                <i
                  class="fas fa-heart"
                />
                 
                <a
                  href="https://www.tsyacom.cl"
                >
                  Tsyacom
                </a>
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </footer>
</DocumentFragment>
`;
